From c55ca4176a4d7dccb4d81999f89349bda1fe2949 Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Date: Thu, 25 Sep 2014 16:07:57 +0300
Subject: [PATCH 323/456] v4l: xilinx: hls: Remove axi- prefix from DT
 properties

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

The prefix has been removed from the compatible strings, remove it from
the video-width and video-format properties as well.

Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
(cherry picked from commit 3d67b632fde0ad7edbe8797972643972ea025d63)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../bindings/media/xilinx/xlnx,axi-hls.txt         |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-hls.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-hls.txt
index 79fa6f5..a13fb58 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-hls.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-hls.txt
@@ -26,12 +26,12 @@ Required properties:
 
 Required port properties:
 
-- xlnx,axi-video-format: Video format as defined in video.txt.
-- xlnx,axi-video-width: Video width as defined in video.txt.
+- xlnx,video-format: Video format as defined in video.txt.
+- xlnx,video-width: Video width as defined in video.txt.
 
 Example:
 
-	axi_hls_0: axi_hls@43c00000 {
+	hls_0: hls@43c00000 {
 		compatible = "xlnx,axi-hls-sobel", "xlnx,axi-hls";
 		reg = <0x43c00000 0x24>, <0x43c00024 0xa0>;
 		clocks = <&clkc 15>;
@@ -43,8 +43,8 @@ Example:
 			port@0 {
 				reg = <0>;
 
-				xlnx,axi-video-format = "yuv422";
-				xlnx,axi-video-width = <8>;
+				xlnx,video-format = "yuv422";
+				xlnx,video-width = <8>;
 
 				hls0_in: endpoint {
 					remote-endpoint = <&vdma_out>;
@@ -53,8 +53,8 @@ Example:
 			port@1 {
 				reg = <1>;
 
-				xlnx,axi-video-format = "yuv422";
-				xlnx,axi-video-width = <8>;
+				xlnx,video-format = "yuv422";
+				xlnx,video-width = <8>;
 
 				hls0_out: endpoint {
 					remote-endpoint = <&vdma_in>;
-- 
1.7.5.4

